### ISSUE TITLE
Aimbot might be better

### ISSUE THREAD

**Issue by Exerciz** [zbot473/SurvivHacks#178]
Hey,

The Aimbot has gotten worse..no matter if I put it on 1 or 10...
It not aims correctly. It was some time ago a lot better..

Please if you make updates for the Script check the Aimbot and how you could make it more smooth and better, because it's the main thing of it.

**Comment by alexvornoffice** [comment]
Can you describe more precise what is the problem, I made some changes so Aimbot will not hit buildings and metal things, are you referring to this?

**Comment by Exerciz** [comment]
No i mean it aims to slow to an enemy.. and the aim not correctly hiting when someone is moving..all shots gets missed..

**Comment by alexvornoffice** [comment]
yes but people started to play better and I think using mobile phones they move faster! 
People increased their play experience and now they dodge bullets very well in comparison some time ago... auto aim need improvement anyway... specially for short range too.

**Comment by alexvornoffice** [comment]
In summer almost every 3rd game was a win, not like one in 10 LOL
